components/radiobuttons/layout.scss

```diff
@@ -0,0 +1,47 @@
+@import "../../lib/interface";
+@import "../../lib/units";
+@import "../../lib/colors";
+
+.mmf-radio-btn-container {
+    height: 0.75 * $mmf-block;
+    display: inline-block;
+}
+
+.mmf-radio-btn {
+    @include clickable();
+    border: $mmf-input-border--width solid transparent;
+    border-radius: $mmf-radius;
+    display: inline-flex;
+    flex-direction: row;
+    align-items: center;
+    margin: 0 $mmf-unit / 2;
+
+    &[disabled] {
+        opacity: $mmf-opacity--disabled;
+    }
+
+    .mmf-label {
+        @include clickable();
+        display: inline-block;
+        padding: $mmf-unit/2 $mmf-unit;
+        color: inherit;
+    }
+
+    .mmf-icon {
+        padding-left: $mmf-unit;
+    }
+
+    &:not([disabled]):hover {
+        border: $mmf-input-border--width solid map-get($color-highlight, 5);
+        border-radius: $mmf-radius;
+        background-color:map-get($color-highlight, 5);
+        color: map-get($color-highlight, 0);
+    }
+
+    &.selected {
+        border: $mmf-input-border--width solid map-get($color-highlight, 5);
+        background-color:map-get($color-highlight, 5);
+        border-radius: $mmf-radius;
+        color: map-get($color-highlight, 0);
+    }
+}
```

Comment:
By convention we usually keep a last empty line within each file to maintain support for line-based shell-tools

Reply:
It seems you dont have a plugin for .editorconfig files. There you can find the option for all files:

```
insert_final_newline = true
```